app/models/front_end_builds/build.rb

```diff
@@ -8,6 +8,7 @@ class Build < ActiveRecord::Base
                       :endpoint
     end
 
+    has_one :live_app, class_name: "FrontEndBuilds::App", foreign_key: :live_build_id
```

Comment:
I used it in a test below, to specify which build was live

    let!(:live_build) { FactoryGirl.create :front_end_builds_build, :fetched, app: app, live_app: app }

but I could also just as simply do app.live_build = live_build. So, that's fine.

Is it customary to not have inverse relationships always defined?